migrations/postgres/20250317192000_alias_writer.sql

```diff
@@ -0,0 +1,16 @@
+/* ------------------------------ */
+
+INSERT INTO outbox_message_consumptions (consumer_name, producer_name, last_consumed_message_id)
```

Comment:
Hm, it seems so easy to forget to add such migration and end up replaying months of events upon deployment.

Mb let's cut a ticket to extend consumer interface with a mandatory method that defines how it behaves upon creation - whether it wants to replay the whole history or wants to start listening only to new events?

Reply:
We have such a ticket since October - https://github.com/kamu-data/kamu-cli/issues/929.
Cutting tickets does not help. Actually doing tickets may probably shift the situation.